apps/central-scan/backend/src/fujitsu_scanner.ts

```diff
@@ -15,9 +15,6 @@ const debug = makeDebug('scan:scanner');
 
 export interface BatchControl {
   scanSheet(): Promise<SheetOf<string> | undefined>;
-  acceptSheet(): Promise<boolean>;
-  reviewSheet(): Promise<boolean>;
-  rejectSheet(): Promise<boolean>;
```

Comment:
If there's a reason to keep these around that I don't know about, please do tell, but it seemed like leftover code to me.